library/node

```diff
@@ -95,7 +95,7 @@ Directory: 6.11/wheezy
 
 Tags: 4.8.4, 4.8, 4, argon
 Architectures: amd64, ppc64le
-GitCommit: 9c25cbe93f9108fd1e506d14228afe4a3d04108f
+GitCommit: d001b4cfef7e2d5fede74ce3c6f5e356905ff09c
```

Comment:
This is changed because of an added `architectures` file to _not_ get s390x for 4.8. No actual changes